readme.md

```diff
@@ -96,17 +100,16 @@ const sum = atom((get) => atoms.map(get).reduce((acc, count) => acc + count))
 You can make the read function an async function too.
 
 ```jsx
-const urlAtom = atom("https://json.host.com")
-const fetchUrlAtom = atom(
-  async (get) => {
-    const response = await fetch(get(urlAtom))
-    return await response.json()
-  }
-)
+const urlAtom = atom('https://json.host.com')
+const fetchUrlAtom = atom(async (get) => {
+  const response = await fetch(get(urlAtom))
+  return await response.json()
+})
 
 function Status() {
   // Re-renders the component after urlAtom changed and the async function above concludes
   const [json] = useAtom(fetchUrlAtom)
+}
```

Comment:
This one looks a bit strange.

Reply:
That seems over-complicated. We want to keep readme very compact, and move detailed things into `./docs/`.

Reply:
Got it  , thanks .